pkg/opaqueports/defaults.go

```diff
@@ -5,12 +5,4 @@ package opaqueports
 // When a pod or service already has its own annotation, that value will have
 // priority of this.
 //
-// Note: Keep in sync with proxy.opaquePorts in values.yaml
-var DefaultOpaquePorts = map[uint32]struct{}{
-	25:    {},
-	443:   {},
-	587:   {},
-	3306:  {},
-	5432:  {},
-	11211: {},
-}
+var DefaultOpaquePorts = map[uint32]struct{}{}
```

Comment:
Should we instead completely remove this pkg, and move this field as global variable in `controller`? 🤔 

I started out this way as it was the simple first step. Now trying to simplify this further.

Reply:
Making it a global variable in `controller` was an issue when I first did this because we need to import it in `destination` and `watcher` packages which introduces an import cycle. I moved it into `pkg` package for this reason.

Reply:
the cleaner thing to do would be to pass this as a constructor argument wherever it is needed instead of making it a global at all.

Reply:
Updated. This meant that `opaquePorts map[uint32]struct{}{}` had to be added to `server`, `endpoint_translator` and `opaque_ports_watcher` as each of them use this field induvidually